e2e/join-us.e2e.js

```diff
@@ -2,13 +2,13 @@ import chai from 'chai';
 
 const browser = global.browser; // comes from wdio
 
-describe('Home page', function () {
+describe('Join Us page', function () {
   it('should load', function (done) {
```

Comment:
I'm guilty of doing `it(should…` and ending up with really loooooong spec names too. Present tense sounds good.